manifests/01-namespace.yaml

```diff
@@ -4,8 +4,9 @@ metadata:
   annotations:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
-    openshift.io/node-selector: ""
+    include.release.openshift.io/single-node-developer: "true"
     include.release.openshift.io/single-node-production-edge: "true"
+    openshift.io/node-selector: ""
```

Comment:
can you clarify / remind me the resulting behavior of the node-selector annotation setting above @cfergeau ?

Reply:
/hold 

pending the answer to the above question

Reply:
The `node-selector` annotation is already there before this change, I moved it to group all `include.release.openshift.io` annotations together, but I'm only adding a ` include.release.openshift.io/single-node-developer: "true"` annotation

Before:
```
  annotations:
    include.release.openshift.io/ibm-cloud-managed: "true"
    include.release.openshift.io/self-managed-high-availability: "true"
    openshift.io/node-selector: ""
    include.release.openshift.io/single-node-developer: "true"
```

After:
```
  annotations:
    include.release.openshift.io/ibm-cloud-managed: "true"
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
    include.release.openshift.io/single-node-production-edge: "true"
    openshift.io/node-selector: ""
```

Reply:
ahh ... sorry, missed that ... my bad

/hold cancel

/lgtm